rbd/mirror_nautilus_test.go

```diff
@@ -0,0 +1,398 @@
+//go:build nautilus && ceph_preview
+// +build nautilus,ceph_preview
+
+package rbd
+
+import (
+	"os"
+	"testing"
+	"time"
+
+	"github.com/pkg/errors"
```

Comment:
This is archived, and shouldn't be used anymore. Any reason you want to use this? Is the standard `fmt.Errorf` not sufficient?